takari-lifecycle-plugin/src/main/java/io/takari/maven/plugins/compile/ProjectClasspathDigester.java

```diff
@@ -103,7 +100,7 @@ private boolean digest(String key, List<File> dependencies) {
 
     context.setAttribute(key, new ArrayList<>(digest.values()));
 
-    log.debug("Analyzed {} classpath dependencies ({} ms)", dependencies.size(), stopwatch.elapsed(TimeUnit.MILLISECONDS));
+    log.debug("Analyzed {} classpath dependencies ({} ms)", dependencies != null ? dependencies.size() : 0, System.currentTimeMillis() - started);
 
```

Comment:
Elapsed time shouldn't be measured with wall clock, but `#nanoTime()` only.

Reply:
I agree, but it is for logging purposes only (human consumption), and we, as anyone can tell, eat wall clock as well :smile:

Reply:
Accepted